metadata.yaml

```diff
@@ -0,0 +1,19 @@
+name: training-operator
+display-name: Training Operator
+summary: Tools for distributed or non-distributed training on Kubernetes
+description: |
+  Training Operator provides Kubernetes custom resources that make it
+  easy to run distributed or non-distributed training jobs on Kubernetes.
+  This charm deploys the Operator configured for use with Kubeflow to
+  Kubernetes models in Juju.
+maintainers: [Juju Developers <juju@lists.ubuntu.com>]
+tags: [ai, bigdata, kubeflow, machine-learning, tensorflow, pytorch]
```

Comment:
This is also essentially a no-op for Charmhub -- this field used to be read by the old charm store.

```suggestion
```

Reply:
+1 on the suggestion, thanks